neoemf-core/src/main/java/fr/inria/atlanmod/neoemf/config/ConfigFile.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2013-2017 Atlanmod, Inria, LS2N, and IMT Nantes.
+ *
+ * All rights reserved. This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License v2.0 which accompanies
+ * this distribution, and is available at https://www.eclipse.org/legal/epl-2.0/
+ */
+
+package fr.inria.atlanmod.neoemf.config;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.io.Writer;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Map;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+import javax.annotation.Nonnull;
+import javax.annotation.ParametersAreNonnullByDefault;
+
+import static fr.inria.atlanmod.commons.Preconditions.checkNotNull;
+
+/**
+ * A simple file-based configuration.
+ */
+@ParametersAreNonnullByDefault
+// TODO Manage the type of values: if a long is writen in the file, it should be returned as a long
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)